corehq/apps/reports/templates/reports/report_email.html

```diff
@@ -19,8 +19,10 @@
 <body>
     {% if email %}
     <p>
+        {% trans "This is a" %}
+        {% if once %}{% trans "once off" %}{% else %}{% trans "scheduled" %}{% endif %}
```

Comment:
this type of translation building requires a very specific mapping of english onto other languages that often doesn't work (e.g. if the clauses need to be interspersed or the order reversed).

I think the correct way to do it would be to pass the translation of the fully qualified noun (so "once off report" or "scheduled report")  in as a variable, and then template that with `blocktrans`.